kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/listener/PipelineContextManagerLifecycleListener.java

```diff
@@ -51,6 +61,22 @@ public void onInitialized(final String databaseName, final ContextManager contex
         PipelineContextManager.putContext(contextKey, new PipelineContext(modeConfig, contextManager));
         PipelineMetaDataNodeWatcher.getInstance(contextKey);
         ElasticJobServiceLoader.registerTypedService(ElasticJobListener.class);
+        dispatchEnablePipelineJobStartEvent(contextKey);
+    }
+    
+    private void dispatchEnablePipelineJobStartEvent(final PipelineContextKey contextKey) {
```

Comment:
Is there possible exception thrown in this method?

Reply:
Add runtime exception cache,  It's not thrown exception usually.